lib/byron/test/data/cardano-node/node.config

```diff
@@ -12,6 +12,7 @@ NodeId:
 NumCoreNodes: 1
 PBftSignatureThreshold: 1000000000000000
 Protocol: RealPBFT
+GenesisFile: genesis.yaml
```

Comment:
genesis has to a JSON file though, or at least, it used to be. We'll to inject a path to the genesis file at runtime, like we do for the start time in the genesis.yaml ---> https://github.com/input-output-hk/cardano-wallet/blob/master/lib/byron/test/integration/Cardano/Wallet/Byron/Config.hs#L159-L161

Reply:
Yeah, but I think we'll be fine with a relative `genesis.json`?

Reply:
No, not really... Right now it might works because the PR removing the `--genesis-file` override from the command-line is not part of 1.8.0 but after that, the genesis file is actually generated dynamically and put in temporary location. The node isn't started from this same temporary directory thus it won't probably find the genesis file if specified relatively. 

Your change is however sufficient to get us going with 1.8.0.

Reply:
> `--genesis-file`

Right, that explains why it works.